test/rules/invalid_case_patterns_test.dart

```diff
@@ -17,6 +17,7 @@ class InvalidCasePatternsTest extends LintRuleTest {
   @override
   String get lintRule => 'invalid_case_patterns';
 
+  @FailingTest(reason: 'error is produced with analyzer 5.8.0')
```

Comment:
Can you open an issue for these? What's the error? I'm guessing we want to fix these for the beta?

Reply:
https://github.com/dart-lang/linter/issues/4166

Reply:
Cool!  Can you update the annotations w/ a back pointer? So like:

`@FailingTest(issue: 'https://github.com/dart-lang/linter/issues/4166')`

?

Reply:
Done